src/OgAccess.php

```diff
@@ -116,6 +123,14 @@ public function userAccess(EntityInterface $group, $operation, AccountInterface
       }
     }
 
+    // Update group special permission. At this point, the operation should have
+    // already been handled by Og. If the operation is simply 'edit'
+    // (or 'update' for content entities), it is referring to the current group,
+    // so we have to map it to the special permission.
+    if (in_array($operation, ['update', 'edit'])) {
+      $operation = OgAccess::UPDATE_GROUP_PERMISSION;
+    }
```

Comment:
The 'edit' operation is not supported by any of the core modules I believe, this operation is always called 'update'. Did you see this in a contrib module?